version/version.go

```diff
@@ -19,9 +19,11 @@ import (
 	"runtime"
 )
 
+//var needs to be used instead of const for ldflags
 var (
-	Version    = "v0.15.0+git"
-	GitVersion = "unknown"
-	GitCommit  = "unknown"
-	GoVersion  = fmt.Sprintf("%s %s/%s", runtime.Version(), runtime.GOOS, runtime.GOARCH)
+	Version                 = "v0.14.0+git"
+	GitVersion              = "unknown"
+	GitCommit               = "unknown"
+	KubernetesVendorVersion = "unknown"
```

Comment:
Maybe just `KubernetesVersion` to align with what we output with `operator-sdk version`?